mne/annotations.py

```diff
@@ -810,64 +844,31 @@ def __init__(
         duration,
         description,
         hed_strings,
```

Comment:
If this is a single segment of data with a single duration, I am not sure why there is an array of HED strings rather than a single string -- consisting of multiple (possibly parenthesized) comma separate HED tags?